### PR TITLE
Milestone #2:Setup and Mobile Version Skeleton

### PR DESCRIPTION
**Pull request by Enekwestanley**
This is a project to build a section of a portfolio page.
Technologies used;

-    Github
-    VS code
Built with;
-    HTML
-    CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            </li>
+            <li>
+              <a href="#About">About</a>
+            </li>
+            <li>
+              <a href="#Contact">Contact</a>
+          </li>
+         </ul>
+      </div>
+    </header>
```
Comment (aimemalaika) [inline index.html:29]: - [ ] Consider putting the header out of the section.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <div class="text1">
+      <h1>I'm Stanley Enekwe, Glad to see you!</h1>
+      <p class="header-text">I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
```
Comment (aimemalaika) [inline index.html:35]: - [ ] Consider increasing the `line-height` to increase spacing between lines

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @Enekwestanley , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Highlights
- Linters passed ✅ 
- Best use of flexbox ✅ 
- Nice looking UI ✅ 

## Required Changes ♻️

- [ ] check the required changes below

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+.union {
+  margin: 1rem 3rem 0 0;
+}
+
+h1 {
+  font-size: 40px;
+  margin: 5rem 0 0 4rem;
+}
```
Comment (ShoiraTa) [inline style.css:56]: - [ ] OPTIONAL: For this milestone you are supposed to use `flexbox` to layout items. Using margins is a very bad practice that can lead to numerous issues and bugs in future development. Please consider using `flexbox`.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+.header-text {
+  color: #344563;
+  font-size: 25px;
+  margin: 1rem 0 0 4rem;
+  width: 70%;
+  line-height: 2.5rem;
+}
```
Comment (ShoiraTa) [inline style.css:64]: - [ ] OPTIONAL: Header text looks huge compared to the original design, and the line height is too big, please consider decreasing the font size and line-height to match the design with figma: 

Your design  | Figma design
------------- | -------------
Screenshot 2022-03-17 at 12 11 21 | Screenshot 2022-03-17 at 12 11 54
- [ ] OPTIONAL: Please use the font-family provided in the design which is `Poppins`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <header class="nav1">
+    <a class="logo"  href="#">My logo</a>
+      <div class="nav2">
+        <img class="union" src="img/Union.svg" alt="">
+          <ul class="first-list">
```
Comment (ShoiraTa) [inline index.html:16]: - [ ] OPTIONAL: Humburger icon should be inside `<button>` tag

File: index.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    <div class="connect">
+        <a href="#">LET'S CONNECT</a>
+    </div>
+    <div class="icon1">
+      <ul>
+          <li><a href="#"><i class="icons fab fa-twitter" aria-hidden="true"></i></a></li>
+          <li><a href=""><i class="icons fab fa-linkedin-in" aria-hidden="true"></i></a></li>
+          <li><a href="#"><i class="icons fab fa-medium-m" aria-hidden="true"></i></a></li>
+          <li><a href=""><i class="icons fab fa-github" aria-hidden="true"></i></a></li>            <li><a href="#"><i class="icons fab fa-angellist" aria-hidden="true"></i></a></li>
+      </ul>
```
Comment (ShoiraTa) [inline index.html:46]: - [ ] OPTIONAL: Please consider decreasing the size of buttons and changing color to the provided one which is `#505F79`:

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @Enekwestanley , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

### OPTIONAL Suggestions

_Please check comments below_ 

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**